web/routing.py

```
@@ -0,0 +1,8 @@
+from django.urls import re_path
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)